src/cephadm/cephadm

```diff
@@ -483,6 +485,14 @@ class CephIscsi(object):
                   "umount {0}; fi".format(mount_path)
         return cmd.split()
 
+    def get_tcmu_runner_container(self):
+        # type: () -> CephContainer
+        tcmu_container = get_container(self.fsid, self.daemon_type, self.daemon_id)
+        tcmu_container.entrypoint = '/usr/bin/tcmu-runner'
+        tcmu_container.args.append("/usr/bin/tcmu-runner")
```

Comment:
For some reason in my initial testing having tcmu-runner as the entry point didn't work, but if I added it as an arg it did. But that doesn't make sense, so I plan on testing this again.